src/main/webapp/app/iris/course-chatbot/course-chatbot.component.html

```diff
@@ -0,0 +1,3 @@
+@if (courseId) {
```

Comment:
I guess in the future, there will be additional information here? Otherwise we can just remove this component and put the `@if` in its parent.

Reply:
The component still configures the chat service correctly though